mpmg/src/crawler.py

```diff
@@ -3,60 +3,102 @@
 import sys
 import os
 
-base_url = 'https://transparencia.mpmg.mp.br/db_form/contracheque/'
+base_url = "https://transparencia.mpmg.mp.br/db_form/contracheque/"
```

Comment:
Você pode ter problemas para chamar essa URL de dentro do conteiner por causa do HTTPS

Para guardar para o futuro quando for propor o Dockerfile: https://github.com/dadosjusbr/coletores/blob/master/mprs/Dockerfile#L4